mailer.js

```diff
@@ -272,6 +273,43 @@ module.exports = function (log) {
     })
   }
 
+  Mailer.prototype.unblockCodeEmail = function (message) {
+    log.trace({ op: 'mailer.unblockCodeEmail', email: message.email, uid: message.uid })
+
+    var templateName = 'unblockCodeEmail'
+    var query = {
+      unblockCode: message.unblockCode,
+      email: message.email,
+      uid: message.uid
+    }
+
+    var links = this._generateLinks(null, message.email, query, templateName)
+
+    return this.send({
+      acceptLanguage: message.acceptLanguage,
+      email: message.email,
+      headers: {
+        'X-Uid': message.uid,
+        'X-Unblock-Code': message.unblockCode,
+        'X-Report-SignIn-Link': links.reportSignInLink
+      },
+      subject: gettext('Firefox Account authorization code'),
```

Comment:
I assume you've already worked on this with @ryanfeeley, but it stuck out at me: no title case for a subject?